tests/foreman/performance/common/perf_concurrent_test.py

```diff
@@ -0,0 +1,372 @@
+"""Test utilities for writing performance tests
+
+Define ConcurrentTestCase as parent class of writting Subscribe by AK,
+Subscribe by register and attach, Subscription Deletion.
+
+"""
+
+from robottelo.common import ssh
```

Comment:
Organize the imports in blocks, first the import statements then `from ... import ...`:

```python
import a
import b

from x import y
from z import w
```